README.adoc

```diff
@@ -163,14 +163,17 @@ include::{common-includes}/trywhatyoubuild-end.adoc[]
 
 Navigate to the `start` directory.
 
-Begin by building your response JSON, which is composed of the `_links` array, as well as the name of
+[role=command]
+include::{common-includes}/devmode-start.adoc[]
+
+Begin by building your response JSON, which is composed of the `_links` array and the name of
 the host machine.
 
 === Linking to inventory contents
 
 As mentioned before, your starting point is an existing simple inventory REST service. 
 
-Take a look at the `request handlers` in the `src/main/java/io/openliberty/guides/microprofile/InventoryResource.java` file.
+Take a look at the `request handlers` in the [file=0]`src/main/java/io/openliberty/guides/microprofile/InventoryResource.java` file.
```

Comment:
The hotspot for `src/main/java/io/openliberty/guides/microprofile/InventoryResource.java`
needs to be `[hotspot file=0]`, not `[file=0]`